app/assets/javascripts/angular/helpers/sc2.js

```diff
@@ -271,7 +275,11 @@ Sc2.timeToFrame = function(time) {
   parts = time.split(':');
   seconds = parseInt(parts[parts.length-1]);
   minutes = parseInt(parts[parts.length-2]);
-  frame = ((minutes*60) + seconds) * 16;
+  fps = 16;
+  if (expansion >= 2) {
+    fps *= 1.4;
```

Comment:
i would recommend using a constant for the 1.4, because i suspect we may want to change it to 1.36 or 1.38 at some point, and we want it to be easy to change everywhere.